pkg/cmd/files/find/find.go

```diff
@@ -87,3 +109,18 @@ func RunE(f file.IFinder) func(_ *cobra.Command, args []string) error {
 		return nil
 	}
 }
+
+func AssertFlagsAreValid() (bool, error) {
+	lockfileOnly := viper.GetBool(LockfileOnlyFlag)
+	strict := viper.GetInt(StrictFlag)
+
+	if lockfileOnly && strict != 0 {
+		return false, errors.New("'lockfile' and 'strict' flags are mutually exclusive")
+	}
+
+	if strict < 0 || strict > 2 {
+		return false, errors.New("'strict' supports values within range 0-2")
+	}
+
+	return true, nil
+}
```

Comment:
`lockFileOnly` and `strict` are both already declared vars in the package. Here you are overwriting whatever values they might have. I don't think that is optimal.

Also, here you only return `true` when there is no error. Hence, it is the same as returning an error or not. To keep the returned statements as few as possible, I think this method shall only return error